core/core.go

```diff
@@ -591,6 +592,23 @@ func (cc *ChainCode) Invoke(stub shim.ChaincodeStubInterface) (r peer.Response)
 			span.SetStatus(codes.Error, errMsg)
 			return shim.Error(errMsg)
 		}
+
+	case ExecuteTasks:
+		bytes, err := TaskExecutorHandler(
```

Comment:
Tasks vs Task

Reply:
I prefer to call the service for execution task in the singular 'TaskExecutor' and NOT in the plural 'TasksExecutor'

like UserService is not UsersService

Do you agree?

Reply:
It seems to me that this service is for handling multiple tasks, not just one.

Reply:
fixed change handler method name to TasksExecutorHandler